sources/be/wal/bosa-region-wallonia-fr.json

```diff
@@ -0,0 +1,42 @@
+{
+    "data": "https://opendata.bosa.be/download/best/openaddress-bewal.zip",
+    "protocol": "http",
+    "coverage": {
+         "ISO 3166": {
+             "alpha2": "BE-WAL",
+             "state": "Walloon Region",
+             "country": "Belgium"
+         },
+         "country": "be",
+         "state": "walloon region"
+    },
+    "conform": {
+        "format": "csv",
+        "srs": "EPSG:4326",
+        "encoding": "utf-8",
+        "accuracy": 3,
+        "number": "house_number",
+        "street": "streetname_fr",
+        "city":  "muncipality_name_fr",
+        "postcode": "postcode",
+        "id": {
+            "function": "join",
+            "fields": ["region_code","address_id"],
+            "separator": ":"
+        },
+        "unit": "box_number",
+        "lon": "EPSG:4326_lon",
+        "lat": "EPSG:4326_lat"
+    },
+    "compression": "zip",
+    "language": "fr",
+    "website": "https://opendata.bosa.be",
+    "license": {
+        "url": "https://creativecommons.org/licenses/by/4.0"
```

Comment:
since this is CC BY 4.0, we can set "text" to "CC BY 4.0", set attribution: true, and provide an "attribution name" per https://github.com/openaddresses/openaddresses/blob/master/CONTRIBUTING.md

Reply:
ok, done